### PR TITLE
ISSUE#EPR-40: Send Notification to manager and self registred User

### PR DESCRIPTION
**Pull request by aishwa8141**
1)css changes in admin page and filters

### CODE REVIEW

File: ui-sample/util-service/EPRFunctions.js [hunk 1]
```diff
@@ -111,8 +118,12 @@ class EPRFunctions extends Functions {
      */
     sendNotificationForSuccessfullyOnboarding(callback) {
         if(this.request.body.request[entityType].isActive) {
+            if(this.request.body.request[entityType].orgName == 'EkStep') {
+                this.addToPlaceholders('templateId', "onboardSuccessTemplate");
```
Comment (indrajra) [inline ui-sample/util-service/EPRFunctions.js]: Check - looks swapped code.

File: ui-sample/util-service/EPRFunctions.js [hunk 2]
```diff
@@ -161,6 +172,44 @@ class EPRFunctions extends Functions {
         });
     }
 
+    /**
+     * sends notification to manager to onBoard new Employee (self registeration)
+     * @param {*} callback 
+     */
+    sendNotificationForManager(callback) {
+        this.addToPlaceholders('subject', "Request to Onboard " + this.request.body.request[entityType].name)
```
Comment (indrajra) [inline ui-sample/util-service/EPRFunctions.js]: Mark this as subject line - [Action Required] Validate reportee <name>

Comment (aishwa8141) [reply ui-sample/util-service/EPRFunctions.js]: done

File: ui-sample/util-service/EPRFunctions.js (hunk 1 above)
Comment (aishwa8141) [reply ui-sample/util-service/EPRFunctions.js]: done

**Pull request merged**